docs/docs/tutorial/chapter1/prerequisites.md

```diff
@@ -47,16 +47,17 @@ Please do upgrade accordingly. Then proceed to the Redwood installation when you
 There are many ways to install and manage both Node.js and Yarn. If you're installing for the first time, we recommend the following:
 
 **1. Yarn**
-We recommend following the [instructions via Yarnpkg.com](https://classic.yarnpkg.com/en/docs/install/).
+We recommend following the [instructions via Yarnpkg.com](https://yarnpkg.com/getting-started/install).
 
 **2. Node.js**
-Using the recommended [LTS version from Nodejs.org](https://nodejs.org/en/) is preferred, as the latest Current version isn't supported.
+Using the recommended [LTS version from Nodejs.org](https://nodejs.org/en/) is preferred.
 
 - `nvm` is a great tool for managing multiple versions of Node on one system. It takes a bit more effort to set up and learn, however. Follow the [nvm installation instructions](https://github.com/nvm-sh/nvm#installing-and-updating). (Windows users should go to [nvm-windows](https://github.com/coreybutler/nvm-windows/releases)). For **Mac** users with Homebrew installed, you can alternatively use it to [install `nvm`](https://formulae.brew.sh/formula/nvm).
- **Windows:** Recommended Development Setup
 
-JavaScript development on Windows has specific requirements in addition to Yarn and npm. Follow our simple setup guide:
+**Windows:** Recommended Development Setup
 
-- [Recommended Windows Development Setup](../../how-to/windows-development-setup.md)
+- JavaScript development on Windows has specific requirements in addition to Yarn and npm. Follow our simple setup guide:
+
+  [Recommended Windows Development Setup](../../how-to/windows-development-setup.md)
```

Comment:
These changes are just for better formatting; here's how it renders on the site now:

<img width="948" alt="image" src="https://user-images.githubusercontent.com/32992335/235028829-8e549eaf-017a-4ff4-a5f6-76f5db45e980.png">